### PR TITLE
YYFI LOGO UPLODE

### PR DESCRIPTION
**Pull request by yyfiprotocol**

### CODE REVIEW

**Comment by yyfiprotocol** [comment]
https://explorer.binance.org/tx/215BBA5BE88A1287C2D7D1EFCA2FAF0E0CA5524251C148555CFF458F158DF2F2

**Comment by yyfiprotocol** [comment]
0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a logo

**Comment by yyfiprotocol** [comment]
Hello sir Plasse add my logo trust wallet

**Comment by catenocrypt** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 100 holder addresses and 500 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved
This token has no usage yet (1).
Putting this PR on hold.  Please come back when all criteria are met.

https://etherscan.io/token/0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a

**Comment by yyfiprotocol** [comment]
Hello sir

**Comment by yyfiprotocol** [comment]
Sir, please add the logo of my token

**Comment by yyfiprotocol** [comment]
@zachzwei sir Plasse add my token logo

**Comment by yyfiprotocol** [comment]
Sir my web site link web site Sir the developer team to create the website
https://yyfiprotocol.com/
[yyfiwhitepaper.pdf](https://github.com/trustwallet/assets/files/5181229/yyfiwhitepaper.pdf)
Chack plasse sir Plasse add my token logo trust wallet

**Comment by yyfiprotocol** [comment]
@zachzwei @catenocrypt @rfunduk @corymsmith  all sir Plasse add my logo

**Comment by yyfiprotocol** [comment]
Sir, please add my token logo
0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a logo

**Comment by yyfiprotocol** [comment]
Add Token Logo

•Token:YYFi.Protocol
•Name: YYFI
•Network: Etherscan-Erc20 
•Contract: 0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a
•Explorer:https://etherscan.io/tx/0x3976c83011029d9f9a0f6710c39b2fecb4d4aeb60099ca3e8e862392194c6bc2

•Website: yyfiprotocol.com web site make our developer team 
•Email: yyfiprotocol@gmail.com
[yyfiwhitepaper.pdf](https://github.com/trustwallet/assets/files/5181308/yyfiwhitepaper.pdf)

**Comment by yyfiprotocol** [comment]
Sir @zachzwei  add my coin logo
0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a logo

**Comment by yyfiprotocol** [comment]
Sir Plasse add my token logo   https://explorer.binance.org/tx/215BBA5BE88A1287C2D7D1EFCA2FAF0E0CA5524251C148555CFF458F158DF2F2
https://etherscan.io/tx/0x3976c83011029d9f9a0f6710c39b2fecb4d4aeb60099ca3e8e862392194c6bc2

**Comment by yyfiprotocol** [comment]
Sir Plasse add my logo 
0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a logo

**Comment by yyfiprotocol** [comment]
@zachzwei sir Plasse add my logo

**Comment by zachzwei** [comment]
Please provide more details about this project. And it should meet the requirements mentioned here:
https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved

You can follow this format when adding an info.json file:

``` {
   "name":"Bitcoin",
   "website":"https://bitcoin.org",
   "source_code":"https://github.com/bitcoin",
   "white_paper":"https://bitcoin.org/bitcoin.pdf",
   "short_description":"Bitcoin is a cryptocurrency and worldwide payment system.",
   "explorer":"https://blockchain.info",
   "research":"https://research.binance.com/en/projects/bitcoin"
}

**Comment by yyfiprotocol** [comment]
Oky sir

**Comment by yyfiprotocol** [comment]
Sir info.json file uplode done

**Comment by yyfiprotocol** [comment]
Sir Plasse add my logo info.json file uplode done @zachzwei

File: blockchains/ethereum/info/info.json [hunk 1]
```diff
@@ -1,22 +1,7 @@
-{
```
Comment (yyfiprotocol) [inline blockchains/ethereum/info/info.json:1]: "name":"YYFi.Protocol",
   "website":"https://yyfiprotocol.com/",
   "source_code":"https://github.com/yyfiprotocol",
   "white_paper":"https://yyfiprotocol.com/yyfiwhitepaper.pdf",
   "short_description":"yyfi.protocol is the largest global Digital Currency Based on Crypto Network.",
   "explorer":"https://etherscan.io/tx/0x3976c83011029d9f9a0f6710c39b2fecb4d4aeb60099ca3e8e862392194c6bc2",

**Comment by yyfiprotocol** [comment]
@zachzwei sir Plasse add my logo

**Comment by yyfiprotocol** [comment]
@zachzwei @catenocrypt  Plasse sir add my token logo

**Comment by yyfiprotocol** [comment]
"name":"YYFi.Protocol",
   "website":"https://yyfiprotocol.com/",
   "source_code":"https://github.com/yyfiprotocol",
   "white_paper":"https://yyfiprotocol.com/yyfiwhitepaper.pdf",
   "short_description":"yyfi.protocol is the largest global Digital Currency Based on Crypto Network.",
   "explorer":"https://etherscan.io/tx/0x3976c83011029d9f9a0f6710c39b2fecb4d4aeb60099ca3e8e862392194c6bc2",

**Comment by yyfiprotocol** [comment]
Hello sir Plasse add my logo

**Comment by yyfiprotocol** [comment]
Sir Plasse help me

**Comment by yyfiprotocol** [comment]
0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a logo

**Comment by catenocrypt** [comment]
Fixed the files in this PR, they were totally mixed up.

**Comment by catenocrypt** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 100 holder addresses and 500 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved
This token has very low usage (4).  Putting PR on hold.  Please come back when all criteria are met.

https://etherscan.io/token/0xaf20b44c1c651d1d29cfb916ee2a0630b828eb7a

**Comment by yyfiprotocol** [comment]
Sir it is not possible without doing the transaction

**Pull request closed without merging**